examples/ChatQnA/deploy/gaudi.md

```diff
@@ -297,28 +297,28 @@ Set the necessary environment variables to setup the use case case
 
 ### Dataprep
 
-    export DATAPREP_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/dataprep"
-    export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/get_file"
-    export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/delete_file"
+export DATAPREP_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/dataprep"

```

Comment:
When you removed the four-space indent, you also changed this to no longer format it as a "code block".  You should either put these indents back, or use the explicit code-block fence using ``` before and after what should be a code-block.